### PR TITLE
DDF-3480 Coverity Findings - restore missing method

### PR DESCRIPTION
**Pull request by ahoffer**
Restore missing method
  Remove dupliced dependency

#### What does this PR do?
Fixes an error introduced in an earlier merge.
#### Who is reviewing it? 
@anyone 

#### Choose 2 committers to review/merge the PR. 
@bdeining
@millerw8
@mrmateo
#### How should this be tested? (List steps with links to updated documentation)
CI build

#### What are the relevant tickets?
DDF-3480
#### Checklist:
- [N/A] Documentation Updated
- [N/A] Update / Add Unit Tests
- [N/A] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an inline code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

File: catalog/spatial/csw/spatial-csw-source-common/pom.xml [hunk 1]
```diff
@@ -33,10 +33,6 @@
             <groupId>ddf.platform.security</groupId>
             <artifactId>security-rest-cxfwrapper</artifactId>
         </dependency>
-        <dependency>
-            <groupId>ddf.security</groupId>
-            <artifactId>ddf-security-common</artifactId>
-        </dependency>
```
Comment (ahoffer) [inline catalog/spatial/csw/spatial-csw-source-common/pom.xml]: This dependency was duplicated in the POM.

File: catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java [hunk 2]
```diff
@@ -1815,6 +1815,14 @@ protected void addSourceMonitor(SourceMonitor sourceMonitor) {
...
   }
 
+  /** Clean-up when shutting down the CswSource */
+  public void destroy(int code) {
+    LOGGER.debug("{}: Entering destroy()", cswSourceConfiguration.getId());
+    availabilityPollFuture.cancel(true);
+    scheduler.shutdownNow();
+    removeEventServiceSubscription();
+  }
+
```
Comment (ahoffer) [inline catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java]: It turns out there a subclass whose blueprint file registers the method.

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

java.lang.NoClassDefFoundError: org/apache/activemq/artemis/jms/client/ActiveMQConnection$1
 at org.apache.activemq.artemis.jms.client.ActiveMQConnection.close(ActiveMQConnection.java:364) ~[?:?]
 at org.apache.camel.component.sjms.jms.ConnectionFactoryResource.destroyObject(ConnectionFactoryResource.java:130) ~[?:?]
 at org.apache.camel.component.sjms.jms.ConnectionFactoryResource.destroyObject(ConnectionFactoryResource.java:31) ~[?:?]
 at org.apache.commons.pool.impl.GenericObjectPool.destroy(GenericObjectPool.java:1339) ~[?:?]
 at org.apache.commons.pool.impl.GenericObjectPool.clear(GenericObjectPool.java:1324) ~[?:?]
 at org.apache.commons.pool.impl.GenericObjectPool.close(GenericObjectPool.java:1495) ~[?:?]

**Comment by bdeining** [comment]
build now

**Pull request merged**